### PR TITLE
Update Andersen Lab ingest to new metadata

### PR DESCRIPTION
**Pull request by joverlee521**
## Description of proposed changes

Update the Andersen Lab ingest workflow to use the latest metadata file that includes SRA runs from multiple BioProjects.

## Related issue(s)

Flagged by @trvrb on [Slack](https://bedfordlab.slack.com/archives/CD84ELG0N/p1719349165673599) that the Andersen Lab repo has started using a new metadata file that now includes multiple BioProjects.

## Checklist

- [x] Checks pass
- [x] [Trial ingest workflow](https://github.com/nextstrain/avian-flu/actions/runs/9670599759) -> results uploaded to `s3://nextstrain-data/files/workflows/avian-flu/trial/update-andersen/h5n1/`

### CODE REVIEW

File: ingest/build-configs/ncbi/defaults/host-map.tsv [hunk 1]
```diff
@@ -50,6 +50,8 @@ harris hawk	Avian
 harris-hawk	Avian
 hawk	Avian
 Homo sapiens	Human
+HOSP	Environment
```
Comment (joverlee521) [inline ingest/build-configs/ncbi/defaults/host-map.tsv]: I'm assuming `HOSP` stands for "hospital" here. Please chime in if this stands for some obscure host name that I'm not aware of.

Comment (trvrb) [reply ingest/build-configs/ncbi/defaults/host-map.tsv]: HOSP = house sparrow, so this should be `HOSP\tAvian`.

Funnily enough there's a semi-official set of "bird codes". Here's the list: https://www.carolinabirdclub.org/bandcodes.html. May be useful to link from the ingest readme (or from `host-map.tsv`.

Comment (joverlee521) [reply ingest/build-configs/ncbi/defaults/host-map.tsv]: Ah, thank you! Will link to it from the `host-map.tsv`.

**Comment by trvrb** [comment]
Thanks @joverlee521! I just added an update to annotations. This is working as expected for me. With the fix for `HOSP` I think this is good to be merged.

File: ingest/build-configs/ncbi/defaults/host-map.tsv (hunk 1 above)
Comment (joverlee521) [reply ingest/build-configs/ncbi/defaults/host-map.tsv]: Updated in https://github.com/nextstrain/avian-flu/pull/64/commits/1297ecd1d15808ff6e1a7b6a43b4bd0ef943bf62

**Pull request merged**